### PR TITLE
Fail gracefully when camera is undefined

### PR DESCRIPTION
**Pull request by stianjensen**

### CODE REVIEW

File: nin/frontend/app/scripts/services/camera.js [hunk 1]
```diff
@@ -63,7 +63,7 @@
           ]) + ',';
         },
         getCameraLookat: function() {
-          var camera = layer.instance.camera;
+          if (!camera) return;
```
Comment (sigvef) [inline nin/frontend/app/scripts/services/camera.js]: Might as well use layer.instance.camera directly, in case it changes in the layer?

Comment (stianjensen) [inline nin/frontend/app/scripts/services/camera.js]: Can do that. The problem was simply that layer sometimes is undefined

**Comment by stianjensen** [comment]
ping

**Comment by cristeahub** [comment]
lol gracefully :+1:

**Comment by stianjensen** [comment]
https://en.wikipedia.org/wiki/Graceful_exit :wink:

**Comment by cristeahub** [comment]
Nice, didn't know that. But since it doesn't really let the user know what happend I guess it's a "graceful" exit?

**Comment by stianjensen** [comment]
I guess it's actually more like failing silently, then

**Comment by stianjensen** [comment]
Or actually not failing at all, just not doing anything because it doesn't make sense ^^

**Pull request merged**